src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -30,24 +30,50 @@ class JwtAuthenticator implements AuthenticatorInterface
      */
     private $jwt;
 
+    /**
+     * @var string
+     */
+    private $jws;
+
+    /**
+     * @var \DateInterval
+     */
+    private $jwsLifetime;
+
+    /**
+     * @var \DateTime
+     */
+    private $jwsValidTo;
+
     /**
      * Constructor.
      *
-     * @param JwtInterface $jwt
+     * @param JwtInterface  $jwt
+     * @param \DateInterval $jwsLifetime
```

Comment:
Please add the new blank line after last `@param` (before `@throws`). This is code style for easy reading the comment of the method.